test/e2e/kubectl/kubectl.go

```diff
@@ -746,6 +746,23 @@ metadata:
 		})
 	})
 
+	framework.KubeDescribe("Kubectl get componentstatuses", func() {
+		It("should get componentstatuses", func() {
+			By("validating the result")
+			// get component list
+			output := framework.RunKubectlOrDie("get", "componentstatuses", "--no-headers=true")
+			lines := strings.Split(strings.TrimRight(output, "\n"), "\n")
+			// validate result for each component
```

Comment:
better to add message for easy digging like
```
By("getting details of componentstatuses")
```

Reply:
I changed it, and removed comment lines because I thought just enough "By" lines.